pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -34,6 +35,10 @@ import (
 	"k8s.io/kubernetes/pkg/volume"
 )
 
+const (
+	awsElasticBlockStoreUnderProvisioningName = "dummy-for-provisioning"
```

Comment:
How about "placeholder-for-provisioning" instead of "dummy"?

Same with the other 2 volumes.  When an admin queries the PV list, I think "placeholder" is a better word to see than "dummy".